UpSkill_App/UpSkill.Infrastructure/Data/Models/Grade.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UpSkill.Infrastructure.Data.Models
+{
+    public class Grade
+    {
+        [Key]
+        public string Id { get; init; } = Guid.NewGuid().ToString();
```

Comment:
I think we need a

public  float/double Value { get; set; }

Reply:
Yes, I don't know how I missed it. Thank you for the feedback!